File: calculator.py
useAgain = True

while(useAgain):
    num1 = int(input("Enter number 1: "))
    num2 = int(input("Enter number 2: "))

    print("+ | Plus")
    print("- | Minus")
    print("* | Multiply")
    print("/ | Divide")
    option = input("Enter an option: ")

    if option == "+":
        result = num1 + num2
        print(str(num1) + "+" + str(num2) + " is: " + str(result))
    elif option == "-":
        result = num1 - num2
        print(str(num1) + "-" + str(num2) + " is: " + str(result))
    elif option == "*":
        result = num1 * num2
        print(str(num1) + "*" + str(num2) + " is: " + str(result))
    elif option == "-":
        result = num1 / num2
        print(str(num1) + "/" + str(num2) + " is: " + str(result))

    response = input("Would you like to use again? [Y/N]")

    if response == "Y":
        useAgain = True
    elif response == "N":
        useAgain = False

File: calculator.java
import java.util.Scanner;

public class Main {

    public static void main (String[] args) {

        Scanner scanner = new Scanner(System.in);

        //variables
        double num1;
        double num2;
        String option;
        double result;
        boolean useAgain = true;
        String response;

        while(useAgain) {
            response = "";
            result = 0;

            System.out.println("Enter number 1:");
            num1 = scanner.nextDouble();

            System.out.println("Enter number 2:");
            num2 = scanner.nextDouble();

            System.out.println("Enter an option: ");
            System.out.println("+ | Plus");
            System.out.println("- | Minus");
            System.out.println("* | Multiply");
            System.out.println("/ | Divide");

            option = scanner.next();

            switch (option) {
                case "+": result = num1 + num2;
                System.out.println(num1 + " + " + num2 + " is: " + result);
                break;
                case "-": result = num1 - num2;
                System.out.println(num1 + " - " + num2 + " is: " + result);
                break;
                case "*": result = num1 * num2;
                System.out.println(num1 + " * " + num2 + " is: " + result);
                break;
                case "/": result = num1 / num2;
                System.out.println(num1 + " / " + num2 + " is: " + result);
                break;
            }

            // when finished ask user if they would like to use again
            System.out.println("Would you like to use Again? [Y/N]");
            response = scanner.next();

            if (response.equals("Y")) {
                useAgain = true;
            }
            else if (response.equals("N")) {
                useAgain = false;
            }
        }
    }
}
